content/en/docs/tasks/administer-cluster/kms-provider.md

```diff
@@ -95,7 +140,9 @@ Ensure that the KMS plugin runs on the same host(s) as the Kubernetes master(s).
 
 To encrypt the data:
 
-1. Create a new encryption configuration file using the appropriate properties for the `kms` provider:
+1. Create a new encryption configuration file using the appropriate properties for the `kms` provider to encrypt resources like Secrets and ConfigMaps:
+
+  **KMS v1**
```

Comment:
I'd use an actual heading here, and set a fragment identifier similar to the other examples. Or you could use tabs here.